Python/Product/PythonTools/PythonTools/Intellisense/AnalysisVariable.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.PythonTools.Analysis;
+
+namespace Microsoft.PythonTools.Intellisense {
+    internal class AnalysisVariable {
+        public AnalysisVariable(VariableType type, LocationInfo location, int? version = null) {
+            Location = location;
+            Type = type;
+            Version = version;
+        }
+
+        public LocationInfo Location { get; }
+        public VariableType Type { get; }
+        public int? Version { get; }
+    }
+}
```

Comment:
The original implementation also has Equals and GetHashCode. Can we be sure they are not needed?

Reply:
Yes. They are never compared and never used as keys in a dictionary.